### DISCUSSION TITLE
Process checking on a Windows host

### DISCUSSION THREAD

**Discussion by TheDiB** [centreon-archive/centreon-archived#9827] [Q&A]
Hi :)

Since a few weeks I am using Centreon 21.04 with a IT-100 plan at my home lab.
It works really well and I am now trying to add services to my multiple hosts.

However I have a bit of struggle with on of my hosts that runs Windows 10.
All the main services work (CPU, RAM etc...) but I don't understand how to check if a certain process is currently running.

This host is my home Plex Media Server and I would like to check if the "Plex Media Server.exe" process is currently running.
I added an OS-Windows-Process-Generic-SNMP entry :
![image](https://user-images.githubusercontent.com/16703607/118272574-193d4400-b4c3-11eb-905d-2deea8ceebe0.png)

But as you can see, the command constantly returns "no process" : 
![image](https://user-images.githubusercontent.com/16703607/118272821-7802bd80-b4c3-11eb-9a76-6a72b7967ae2.png)

At the host side it is currently running : 
![image](https://user-images.githubusercontent.com/16703607/118272999-aed8d380-b4c3-11eb-8c5d-94353124e888.png)

Can you help me, or send me a few examples of how Windows process probing with Centreon works ? :)

Thanks for your help, and sorry for my awfull English !
Kind regards,
Damien

**Comment by Sims24** [comment]
Hi @TheDiB 

I think this issue is more related to Plugins, feel free to also open one on centreon-plugins Github.

Did you try to use the Discovery feature? Do you see the process in the list? You can reach it through 'Configuration > Services > Scan > Manual'. 

Thanks
Simon
